api/buildstatus/buildstatus_handler.go

```diff
@@ -0,0 +1,151 @@
+package buildstatus
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"log"
+	"sort"
+	"strings"
+
+	"github.com/equinor/radix-api/api/utils"
+	"github.com/equinor/radix-api/models"
+	v1 "github.com/equinor/radix-operator/pkg/apis/radix/v1"
+	"github.com/marstr/guid"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const BUILD_STATUS_SVG_RELATIVE_PATH = "../../badges/build-status.svg"
+
+type BuildStatusHandler struct {
+	accounts models.Accounts
+}
+
+func Init(accounts models.Accounts) BuildStatusHandler {
+	return BuildStatusHandler{accounts: accounts}
+}
+
+// GetBuildStatusForApplication Gets a list of build status for environments
+func (handler BuildStatusHandler) GetBuildStatusForApplication(appName, env string) (*[]byte, error) {
+	var output []byte
+
+	// Get latest RJ
+	serviceAccount := handler.getServiceAccount()
```

Comment:
Just get the serviceAccount directly: serviceAccount := handler.accounts.ServiceAccount
No need to use a function for this.